libnavigation-router/src/main/java/com/mapbox/navigation/route/offboard/RouteBuilderProvider.kt

```diff
@@ -11,9 +11,8 @@ import com.mapbox.navigation.base.internal.extensions.inferDeviceLocale
 internal object RouteBuilderProvider {
 
     fun getBuilder(
-        accessToken: String,
         context: Context,
-        urlSkuTokenProvider: UrlSkuTokenProvider
+        urlSkuTokenProvider: UrlSkuTokenProvider?
     ): MapboxDirections.Builder =
         MapboxDirections.builder()
             .profile(DirectionsCriteria.PROFILE_DRIVING_TRAFFIC)
```

Comment:
`continueStraight(true)` is set to `true` but afterwards is `reset` to `false` if `options.continueStraight()` is not available? Should we remove this or make `MapboxDirections.Builder.routeOptions` `continueStraight` default to `true`?

Reply:
`roundaboutExits(true)` is set to `true` but afterwards is `reset` to `false` if `options.roundaboutExits()` is not available? Should we remove this or make `MapboxDirections.Builder.routeOptions` `roundaboutExits` default to `true`?

Reply:
totally make sense, but out of current PR. Caught to https://github.com/mapbox/mapbox-navigation-android/issues/4174